py/torch_tensorrt/dynamo/lowering/passes/replace_rmsnorm.py

```diff
@@ -0,0 +1,36 @@
+import logging
```

Comment:
This pass should be out of tree (i.e. in the example code) and added dynamically via the pass manager